UPGRADE-2.0.md

```diff
@@ -0,0 +1,143 @@
+# Upgrade guide from 1.x to 2.x
+
+## Needed actions
+This is the list of actions that you need to take when upgrading this bundle from the 1.x to the 2.x version:
+
+- Remove `FixturesTrait`:
+    - Access through the service `liip_test_fixtures.services.database_tool_collection` instead
+    - Use `loadAliceFixture(…)` instead of `loadFixtureFiles(…)`
+    - `loadFixtures()` and `loadFixtureFiles()` only accept 2 arguments, here are the old arguments and the new way:
+        - 3rd argument `$omName`: call `$databaseTool->setObjectManagerName($omName);` instead
```

Comment:
is it convenient to set this for operations like fixture loading? since with this flow we need to call it explicitly before any fixture loading (even for default manager), since it's not really known which entity manager was operated on last load. or we need to explicitly `reset`  object manager name after operating on non-default one.

maybe we can have some scope tooling like 
```
$databaseTool->withObjectManager($omName)
  ->loadAliceFixture(...)
```
where `withObjectManager` return preconfigured tool for this OM without really touching global state?

Reply:
Thanks, you're right. I thought switching to a more OOP syntax would make code more readable but it means changing the global state as you said, that would cause more side effects than keeping the current behaviour.

Reply:
@scaytrase do you have ideas or examples to implement something like this?

Reply:
I don't have any particular examples for tools like this, but in general we can involve Builder pattern here combined with the resulting interface itself (like [ContainerBuilder extend Container](https://github.com/symfony/dependency-injection/blob/5.x/ContainerBuilder.php) in symfony)

By default tool is set too... well, defaults (default em, default connection, etc)

The easies way to implement things like this for me is 
```
final class DatabaseToolBuilder implements DatabaseToolBuilderInterface, DataBaseToolInterface {
   private string $omName = 'default';

   public functoin __construct(/* Some dependencies here  like manager registry) {}

   public function withObjectManager(string $omName) {
       $newTool = clone $this;
       $this->omName = $omName;

       return $newTool;
   }

  public function loadAliceFixtures(string $fixture): void {
      // involves $this->omName here
  }
}
```

From container we should get pre configured tool with all dependencies needed (registries, connections, etc), working with defaults OOB.

Reply:
Thanks a lot! I'll try this.

Reply:
I couldn't change the object manager on the fly, I reverted this: https://github.com/liip/LiipTestFixturesBundle/commit/87d77d99ae553edc8da29ba15a9a5d71bcda49e1

The old call: https://github.com/liip/LiipTestFixturesBundle/blob/18e0755bea82164689755edceee8e04aa4c55f83/src/Services/DatabaseToolCollection.php#L49-L57

I think it can be done from the database tool instance, only from the collection, but I have no explanation yet.